src/tests/ftest/security/container_security_acl.py

```diff
@@ -0,0 +1,176 @@
+        acl_file_name = os.path.join(
+            self.tmp, self.params.get(
+                "acl_file_name", "/run/container_acl/*", "cont_test_acl.txt"))
+
+        #(2)Create pool and container with acl
+        self.log.info("(2)Create a pool and a container with acl\n",
+                      "   base_acl_entries= %s\n", base_acl_entries,
+                      "   acl_file_name= %s\n", acl_file_name)
+        self.pool_uuid, self.pool_svc = self.create_pool_with_dmg()
+        secTestBase.create_acl_file(acl_file_name, base_acl_entries)
+        self.container_uuid = self.create_container_with_daos(
+            self.pool, None, acl_file_name)
+
+        #(3)Verify container permissions rw, rw-attribute
+        permission_type = "attribute"
+        self.log.info("(3)==>Verify container permission %s", permission_type)
+        self.update_container_acl(
+            secTestBase.acl_entry(user_type, self.current_user, "rw"))
+        self.verify_cont_rw_attribute(
+            "write", "pass", attribute_name, attribute_value)
+        self.setup_container_acl_and_permission(
+            user_type, self.current_user, permission_type, cont_permission)
+        self.log.info(
+            "(3.1)Verify container_attribute: write, expect: %s", expect_write)
+        self.verify_cont_rw_attribute(
+            "write", expect_write, attribute_name, attribute_value)
+        self.log.info(
+            "(3.2)Verify container_attribute: read, expect: %s", expect_read)
+        self.verify_cont_rw_attribute("read", expect_read, attribute_name)
+
+        #(4)Verify container permissions tT rw-property
+        permission_type = "property"
+        self.log.info("(4)==>Verify container permission tT, rw-property")
+        self.log.info(
+            "(4.1)Update container-acl %s, %s, permission_type: %s with %s",
+            user_type, self.current_user, permission_type, cont_permission)
+        self.setup_container_acl_and_permission(
+            user_type, self.current_user, permission_type, cont_permission)
+        self.log.info(
+            "(4.2)Verify container_attribute: read, expect: %s", expect_read)
+        self.verify_cont_rw_property("read", expect_read)
+        self.log.info(
+            "(4.3)Verify container_attribute: write, expect: %s", expect_write)
+        self.verify_cont_rw_property(
+            "write", expect_write, property_name, property_value)
+        self.log.info(
+            "(4.4)Verify container_attribute: read, expect: %s", expect_read)
+        self.verify_cont_rw_property("read", expect_read)
+
+        #(5)Verify container permissions aA, rw-acl
+        permission_type = "acl"
+        self.log.info("(5)Verify container permission aA, rw-acl ")
+        self.log.info(
+            "(5.1)Update container-acl %s, %s, permission_type: %s with %s",
+            user_type, self.current_user, permission_type, cont_permission)
+        expect = "pass"  #User who created the container has full acl access.
```

Comment:
You could change the owner to a different user in order to verify the case where the user is NOT the owner, when get/set-ACL access is denied.

Reply:
Thanks for the comment.
I have added a new test at step (6-4), after set-owner and verify the "deny" of acl-set.